apps/web/app/dashboard/profile/page.tsx

```
@@ -187,6 +271,25 @@ export default function ProfilePage() {
       </div>
 
+      {/* Validation Errors Display */}
+      {validationErrors.length > 0 && (
+        <div className="bg-red-50 dark:bg-red-900/20 border border-red-200 dark:border-red-800 rounded-lg p-4">
+          <h3 className="text-sm font-semibold text-red-800 dark:text-red-200 mb-2">
+            Please fix the following errors:
+          </h3>
+          <ul className="list-disc list-inside space-y-1">
+            {validationErrors.map((error, index) => (
+              <li
+                key={index}
```

Biome (2.3.13)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)